src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1661,9 +1661,14 @@ public String toString() {
 			result.append(", height: ");
 			result.append(getHeight());
 			result.append(", frame rate: ");
+			if (isNotBlank(frameRate)) {
 			result.append(getFrameRate());
+			} else { result.append("variable"):
+			}
```

Comment:
@Sami32 No, videoBitDepth is an ```int```. Jav differentiates between "primary data types" and "classes". Class-based types use ```equals()```, ```compareTo()``` etc while primary data types use operators like ```==```, ```!=``` etc. Trying to use ```equals()``` there would give a compile error.